cms/standard_pages/models.py

```diff
@@ -38,3 +40,93 @@ class InformationPage(BasePage):  # type: ignore[django-manager-missing]
         index.SearchField("summary"),
         index.SearchField("content"),
     ]
+
+
+class IndexPage(BasePage):  # type: ignore[django-manager-missing]
+    template = "templates/pages/index_page.html"
+
+    parent_page_types: ClassVar[list[str]] = ["home.HomePage", "IndexPage"]
+    subpage_types: ClassVar[list[str]] = ["IndexPage", "InformationPage"]
+
+    description = models.TextField()
```

Comment:
My bad. All our page models use `summary` for this field, so let's be consistent.

This also removes the need for the separate checks like in 10d2cd6